scripts/generate_cedar_intake_client

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found."
+    echo "Install jq from https://stedolan.github.io/jq/download/"
+    exit 1
+fi
+
+if ! command -v swagger &> /dev/null
+then
+    echo "go-swagger could not be found."
+    echo "Install go-swagger from https://goswagger.io/install.html"
+    exit 2
+fi
```

Comment:
Should we add this to the `prereqs` task of `scripts/dev`? Might be good to be able to point this stuff out as part of that script as well.

https://github.com/CMSgov/easi-app/blob/07ba3facc55b7c0fb60aa985ada8ac3fb9db76cd/scripts/dev#L37-L49

Reply:
Added in https://github.com/CMSgov/easi-app/pull/1495/commits/332cc336a0cb9dc0dd0a33488d32a1242e4c142f. I left out the `install_with` because installation differs between Mac and Windows+WSL. I also updated the dev environment setup docs.